composer.json

```diff
@@ -31,7 +31,8 @@
         "symfony/polyfill-intl-icu": "~1.0",
         "symfony/polyfill-mbstring": "~1.0",
         "symfony/polyfill-php56": "~1.0",
-        "symfony/polyfill-php70": "~1.6"
+        "symfony/polyfill-php70": "~1.6",
+        "symfony/polyfill-php80": "~1.17"
```

Comment:
this polyfill is >=7.0.8, so we cannot use it here

Reply:
Oh, I wasn't aware. Thinking about it, code will probably work fine without the polyfill.